README.md

```diff
@@ -14,6 +14,7 @@ These protocols define various components of the [vac](https://vac.dev) stack.
  - [remote log](./remote-log.md) - Remote replication of local logs.
  - [mdf](./mdf.md) - Metadata field for [MVDS](./mvds.md) messages. 
  - [waku](./waku.md) - ÐΞVp2p wire protocol, substituting [EIP-627](https://eips.ethereum.org/EIPS/eip-627).
+ - [wms](./wms.md) - Mailserver specification for archiving and delivering [waku](./waku.md) messages on demand.
```

Comment:
historical waku messages (or envelopes?)

also the `w` in ms might be a good pattern for for envelope as well, so the waku specs are grouped together. Otherwise this list might get a bit bungled